src/SentryBreadcrumbLogTarget.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Sentry;
+
+use Yiisoft\Log\Target;
+
+final class SentryBreadcrumbLogTarget extends Target
+{
+    public function __construct(private SentryLogAdapter $handler)
+    {
+        parent::__construct();
+    }
+
+    /**
+     * @psalm-suppress MixedArgument
+     */
+    protected function export(): void
+    {
+        foreach ($this->getMessages() as $message) {
```

Comment:
Can `breadcrumb` and `log` be called within a single loop?

Reply:
yes can, but it improves readability